flocker/node/agents/test/test_blockdevice.py

```diff
@@ -66,6 +68,18 @@
 from ....common.test.test_thread import NonThreadPool, NonReactor
 
 LOOPBACK_BLOCKDEVICE_SIZE = int(MB(64).to_Byte().value)
+CLEANUP_RETRY_LIMIT = 10
+
+# Helper structures to Eliot output leftover volumes after a test run.
+VOLUMES = Field.for_types(
```

Comment:
Since this is test code only you could (in the future) just use `start_action` directly; typed messages are good for things like production code where we want to write unit tests to validate that logging really works, and do serialization etc.. For logging *by* tests it's not needed.